synapse/handlers/room.py

```diff
@@ -1046,6 +1046,18 @@ async def send(etype: str, content: JsonDict, **kwargs: Any) -> int:
             if power_level_content_override:
                 power_level_content.update(power_level_content_override)
 
+            # override default_power_level_content_override for this room preset, if any
+            default_power_level_content_override = (
+                self.config.room.default_power_level_content_override
+            )
+            if (
+                default_power_level_content_override
+                and default_power_level_content_override.get(preset_config)
+            ):
+                power_level_content.update(
+                    default_power_level_content_override.get(preset_config)
+                )
```

Comment:
If an admin were to configure an override like: `private_chat: { "events": { "com.example.foo" : 0 } }`, the entire default `"events": {"m.room.name": ..., ...}` content from above will be replaced. Is this intended behaviour?

kind of related to https://github.com/matrix-org/matrix-spec/issues/492